delta_filter.cpp

```diff
@@ -135,33 +192,40 @@ DeltaFilter::DeltaData::~DeltaData()
  * exceeds the tolerance then the entire reading is sent.
  *
  * Note, the time used when considering rate is not the current time
- * but the tiem in the readings as the rate referes to the reading rate
+ * but the time in the readings as the rate referes to the reading rate
  * and not real time. The two will be different because of buffering
  * without the services that make up a Fledge instance.
  *
- * @param candidiate	The candidiate reading
+ * @param candidate	The candidate reading
  */
 bool
-DeltaFilter::DeltaData::evaluate(Reading *candidiate,
-				  double tolerance,
-				  struct timeval rate)
+DeltaFilter::DeltaData::evaluate(Reading *candidate,
```

Comment:
Many new parameters are included for evaluate function. Better to add info about parameters in doc string

Reply:
Sure, done.